ui/appui-react/src/appui-react/accudraw/AccuDrawFieldContainer.tsx

```diff
@@ -20,8 +20,8 @@ import { KeyboardShortcutManager } from "../keyboardshortcut/KeyboardShortcut";
 import { AccuDrawSetCompassModeEventArgs, AccuDrawSetFieldFocusEventArgs, AccuDrawSetFieldLockEventArgs, FrameworkAccuDraw } from "./FrameworkAccuDraw";
 import { AccuDrawUiSettings } from "./AccuDrawUiSettings";
 
-import angleIconSvg from "./angle.svg?sprite";
-import distanceIconSvg from "./distance.svg?sprite";
+import angleIconSvg from "./angle.svg";
```

Comment:
I assume in a separate pr we will transition to using svgs from [itwinui-icons](https://itwin.github.io/iTwinUI-icons/)?

Reply:
When I make the change to allow react component icons everywhere, we'll use whatever it available in itwinui. They may or may not have all that we need.